src/blog/v8-release-84.md

```diff
@@ -0,0 +1,83 @@
+---
+title: 'V8 release v8.4'
+author: 'Camillo Bruni, '
+avatars:
+ - 'camillobruni'
+date: 2020-06-30
+tags:
+ - release
+description: 'V8 v8.4'
+tweet: '1257333120115847171'
```

Comment:
Looks like a copy-paste leftover, please set to an empty string as this is not the tweet associated with this release.

Reply:
done.